multigeiger/transmission.cpp

```diff
@@ -17,9 +17,19 @@
 #include "ca_certs.h"
 
 // Hosts for data delivery
-#define MADAVI "https://api-rrd.madavi.de/data.php"
-#define SENSORCOMMUNITY "https://api.sensor.community/v1/push-sensor-data/"
-#define TOILET "https://ptsv2.com/t/rk9pr-1582220446/post"
+
+// use http for now, server operator tells there are performance issues with https.
+#define MADAVI "http://api-rrd.madavi.de/data.php"
```

Comment:
true also?

Reply:
@rexfue do you know? is this the same server or admin and does the same issue apply there?

Reply:
Sorry, I don't know for sure. But I guess, there are the same restrictions.

Reply:
It's a different IP, thus a different server.

I'll remove the comment. Guessing is not helpful here.